docs/explanations/advanced.md

```diff
@@ -93,6 +94,12 @@ When the user pushes an application using a source code directory (with the [`ep
 
 When installing Epinio, the user can choose to use an external S3 compatible storage or let Epinio install Minio on the cluster ([See here how](../howtos/setup_external_s3.md)).
 
+### s3gw
+
+[Upstream project link](https://github.com/aquarist-labs/s3gw)
+
+S3gw is another S3-compatible solution incorporated into Epinio helm-chart which can be used instead of default Minio.
```

Comment:
Instead of in terms of Minio, I'd rather write something like this:
`s3gw is a lightweight s3-compatible service that can be backed by any PVC within a Kubernetes environment`

Reply:
@giubacc This info is not so useful for Epinio and mentioning PVC can be IMO confusing. Would this work for you?
`
S3gw is a lightweight S3-compatible solution available in Epinio helm-chart as an alternative to Minio or external S3 provider.
`

Reply:
it is good!